.meteor/versions

```diff
@@ -131,6 +134,9 @@ ongoworks:security@2.0.1
 ordered-dict@1.0.8
 percolate:migrations@0.9.8
 practicalmeteor:chai@2.1.0_1
+practicalmeteor:loglevel@1.2.0_2
```

Comment:
Curious if we need all these dependencies or they were meant for testing purposes and you forgot to clean them up